cfme/tests/physical_infrastructure/ui/test_physical_overview.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+from cfme.physical.overview import PhysicalOverview
+from cfme.physical.provider.lenovo import LenovoProvider
+from cfme.utils.appliance.implementations.ui import navigate_to
+
+pytestmark = [pytest.mark.tier(3),
+    pytest.mark.provider([LenovoProvider])]
+
+
+def test_physical_overview_page():
+    view = navigate_to(PhysicalOverview, 'All')
+    assert view.is_displayed
+
+
+def test_physical_overview_servers_number(appliance, provider):
+    view = navigate_to(PhysicalOverview, 'All')
+    servers = appliance.collections.physical_servers.all(provider)
```

Comment:
I would switch the order of your statements here, since its possible that an `all()` method call could use the UI and navigate.

Though it may currently use DB, or REST to get the physical server list, the method may change, or be implemented with Sentaku - making this test unstable.

Think about the comparison made here as well - the `all()` method that you called will be getting the number of servers from CFME's database or rest interface. You're then comparing the UI number against that - which ends up being a weak comparison. A much stronger comparison, and test, would be to query the number of servers from the provider's REST interface directly via a wrapanapi call to `provider.mgmt.list_servers()`